ajax/libs/openshare/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "openshare",
+  "filename": "openshare.min.js",
+  "description": "Social sharing for developers",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/OpenShare/openshare.git"
+  },
+  "keywords": [
+    "social",
+    "share",
+    "sharing"
+  ],
+  "author": "Digital Surgeons",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/OpenShare/openshare.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/!(test)*"
+        ]
+      }
+    ]
+  },
```

Comment:
Can we use npm auto-update instead, please?

https://www.npmjs.com/package/openshare

```
tree node_modules/openshare

node_modules/openshare
├── README.md
├── analytics.js
├── async-test.html
├── bower.json
├── count.js
├── dist
│   ├── openshare.js
│   ├── openshare.min.js
│   └── test.js
├── lib
│   ├── countReduce.js
│   ├── dashToCamel.js
│   ├── init.js
│   ├── initializeCountNode.js
│   ├── initializeNodes.js
│   ├── initializeShareNode.js
│   ├── initializeWatcher.js
│   ├── setData.js
│   ├── share.js
│   └── storeCount.js
├── package.json
├── share.js
├── src
│   ├── browser.js
│   ├── index.js
│   ├── modules
│   │   ├── count-api.js
│   │   ├── count-transforms.js
│   │   ├── count.js
│   │   ├── data-attr.js
│   │   ├── events.js
│   │   ├── open-share.js
│   │   ├── share-api.js
│   │   └── share-transforms.js
│   └── test.js
└── test.html

4 directories, 32 files
```